_pages/backup-wallet/seeds-advanced.md

```diff
@@ -74,3 +74,32 @@ Place your paper into a plastic bag with a good seal (vacuum sealed even better)
 
 #### Fireproof Document Bag
 You can use these to protect seed backups, and also to protect your physical hardware wallets.
+
+## Example Backup Configuration
+
+| Wallet | 2 of 3                        | 3 of 5                        |
+|--------|-------------------------------|-------------------------------|
+| #1     | Safe at home                  | Safe at home                  |
+| #2     | Bank or trusted friend/family | Safe at home                  |
+| #3     | Bank or trusted friend/family | Bank or trusted friend/family |
+| #4     | -                             | Bank or trusted friend/family |
+| #5     | -                             | Bank or trusted friend/family |
+
+### Items to include with each wallet
+- With each wallet, include the following:
+  - the wallet's seed, on paper or metal
+  - all the wallets' master public keys[^1] (xpub), on paper
+  - in a storage medium (i.e. SD card, thumb drive):
+    - current version of bitcoin core (all platforms)
+    - current version of Specter Desktop (all platforms)
```

Comment:
While backups of the software is nice-to-have, I don't think it's strictly necessary.

This was a bigger problem in the past when people used non-standard wallets, but p2wsh with output descriptors will be supported forever by many platforms (Nunchuk, Sparrow, Blue wallet, Gordian/Noded, etc).